src/App.css

```diff
@@ -0,0 +1,32 @@
+.App{
+    min-height: calc(100vh - 5rem);
+    /* background-color: var(--color-grey-light-1); */
+}
+.dark-theme .text-intro,
+.dark-theme .login-form {
+  color: #a0aec0;
+  background-color: #2b2f3d;
+}
+.dark-theme .nav-bar {
+  color: #a0aec0;
+  background-color: #212726;
+  background-color: #434047;
```

Comment:
try to use color variables, also there are two background colors given to the `.nav-bar` remove one